### PR TITLE
Add ansible molecule job for edpm_ansible

### PR DESCRIPTION
**Pull request by adrianfusco**
Added `edpm-ansible` under the `openstack-k8s-operators` for testing ansible roles with molecule on https://github.com/openstack-k8s-operators/edpm-ansible

The base image is the following one: ci-operator/config/openstack-k8s-operators/ci-framework/openstack-k8s-operators-ci-framework-master.yaml

### CODE REVIEW

**Comment by adrianfusco** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-edpm-ansible-master-unit

**Comment by adrianfusco** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-edpm-ansible-main-unit

**Comment by adrianfusco** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-edpm-ansible-main-unit

**Comment by adrianfusco** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-edpm-ansible-main-molecule-cont

File: ci-operator/config/openstack-k8s-operators/edpm-ansible/OWNERS [hunk 1]
```diff
@@ -0,0 +1,15 @@
+approvers:
+  - cescgina
+  - cjeanner
+  - rlandy
+  - raukadah
+  - sandeepyadav93
+  - viroel
```
Comment (raukadah) [inline ci-operator/config/openstack-k8s-operators/edpm-ansible/OWNERS:7]: I would suggest to add following members to the list(based on (https://github.com/openstack-k8s-operators/edpm-ansible/graphs/contributors)
- slagle
- rebtoor
- kajinamit
- bshephar
- raukadah
- rabi

**Comment by adrianfusco** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-edpm-ansible-main-molecule-cont

**Comment by adrianfusco** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-edpm-ansible-main-molecule-cont

**Comment by adrianfusco** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-edpm-ansible-main-molecule-cont

**Comment by adrianfusco** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-edpm-ansible-main-molecule-cont

**Comment by adrianfusco** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-edpm-ansible-main-molecule-cont

**Comment by adrianfusco** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-edpm-ansible-main-molecule-test

**Comment by adrianfusco** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-edpm-ansible-main-molecule-test

File: ci-operator/config/openstack-k8s-operators/edpm-ansible/OWNERS (hunk 1 above)
Comment (adrianfusco) [reply ci-operator/config/openstack-k8s-operators/edpm-ansible/OWNERS:7]: Thank you, I've updated it.

**Comment by cjeanner** [comment]
@viroel not sure if we can `/lgtm` it? IIRC there was a discussion a couple of weeks ago about how may or not merge new CI config..

**Comment by cjeanner** [comment]
/lgtm

let's see if I have that power in here :).

File: ci-operator/config/openshift/pagerduty-short-circuiter/openshift-pagerduty-short-circuiter-main.yaml [hunk 2]
```diff
@@ -12,8 +12,24 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: build
```
Comment (viroel) [inline ci-operator/config/openshift/pagerduty-short-circuiter/openshift-pagerduty-short-circuiter-main.yaml:15]: not sure what is happenning, but you made those changes in a config from another repo. Maybe a bad rebase?

File: ci-operator/jobs/openshift/pagerduty-short-circuiter/openshift-pagerduty-short-circuiter-main-presubmits.yaml [hunk 3]
```diff
@@ -1,5 +1,152 @@
 presubmits:
   openshift/pagerduty-short-circuiter:
+  - agent: kubernetes
```
Comment (viroel) [inline ci-operator/jobs/openshift/pagerduty-short-circuiter/openshift-pagerduty-short-circuiter-main-presubmits.yaml:3]: same thing, this is not our file.

**Review by viroel — CHANGES_REQUESTED** [review]
I think that you got a bad rebasing here, since you are changing files from another org/repo.

**Comment by adrianfusco** [comment]
I'll close this PR. It's not possible to use Prow CI due permission problems. @cjeanner  will implement it on Zuul.

**Pull request closed without merging**